experimenter/tests/experimenter_legacy_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+set +x
+
+export PATH=$PATH:/home/seluser/.local/bin
+
+sudo apt-get -qqy update && sudo apt-get -qqy install python3-venv python3-pip
```

Comment:
Isnt this in the dockerfile now?

Reply:
No, we don't control this dockerfile, its built directly from selenium source